### DISCUSSION TITLE
Open an app using custom protocol

### DISCUSSION THREAD

**Discussion by guysiTomax** [SafeExamBrowser/seb-mac#267] [Q&A]
Hello,
I am using SEB for Mac version 3.2.2 and I am having an issue with the start URL.
I need to open a desktop application inside the SEB, and the only solution for this problem I could think of is creating custom protocol "vix://", that opens the application, and passing it to the start URL.
When I try to set "vix://" as the start URL, I get an 'Unsupported URL' error message (checked logs, the exact error is NSURLErrorDomain code=1002).
This problem does not occur when I use this start URL on SEB for windows version 3.3.1.
Thank you for your assistance.

**Comment by danschlet** [comment]
Currently it's not possible to open another app running alongside SEB for macOS. You could open an app using a script before starting SEB, but that app has to run in the background and cannot display any window or panel on top, otherwise SEB will hide it immediately or terminate it if it cannot be hidden.

In a future SEB version for Mac it will be possible to run third party apps in a secured way using the new option in the macOS built-in Assessment Mode (AAC kiosk mode) when using macOS 12 or later.

**Comment by guysiTomax** [comment]
Thank you very much
